packages/harmony/src/components/button/Button.stories.tsx

```diff
@@ -52,3 +83,6 @@ export const Tertiary: Story = { args: { variant: ButtonType.TERTIARY } }
 
 // Destructive
 export const Destructive: Story = { args: { variant: ButtonType.DESTRUCTIVE } }
+
+// Hiding at certain widths
+export const HiddenMobileText: Story = { args: { widthToHideText: 900 } }
```

Comment:
this is not strictly for mobile, right?

Reply:
True, my intent was to give it a name that kind of implied what the use case is, but I think that may be better served for documentation paired with it whenever we do that